src/components/paradox/results/personality/career/modal/resources.js

```diff
@@ -18,7 +21,8 @@ CareerModalResources.propTypes = {
   resources: PropTypes.arrayOf(
     PropTypes.shape({
       name: PropTypes.string.isRequired,
-      description: PropTypes.string.isRequired
+      description: PropTypes.string.isRequired,
+      url: PropTypes.string.isRequired
```

Comment:
I think the URL shouldn't be required (the code above works without it already)

Reply:
Great catch! Ok Removed!